robot/basestation/static/js/elements/feed/navbar.js

```diff
@@ -21,18 +21,17 @@ $(document).ready(() => {
   preferred_theme = getCookie('theme'); // getCookie() from helpers.js
   if (preferred_theme != "") {
       prefer_theme(preferred_theme);
+  }else {
```

Comment:
CS - one space before and after `else`

Reply:
I added a space before `else` but there's already one after it.

Reply:
Fixed